contracts/staking/contracts/src/libs/LibStakingRichErrors.sol

```diff
@@ -295,49 +200,62 @@ library LibStakingRichErrors {
         );
     }
 
-    function InvalidMakerSignatureError(
-        bytes32 poolId,
-        address makerAddress,
-        bytes memory makerSignature
+    function MakerAddressAlreadyRegisteredError(
+        address makerAddress
     )
         internal
         pure
         returns (bytes memory)
     {
         return abi.encodeWithSelector(
-            INVALID_MAKER_SIGNATURE_ERROR_SELECTOR,
-            poolId,
-            makerAddress,
-            makerSignature
+            MAKER_ADDRESS_ALREADY_REGISTERED_ERROR_SELECTOR,
+            makerAddress
         );
     }
 
-    function MakerAddressAlreadyRegisteredError(
-        address makerAddress
+    function MakerAddressNotRegisteredError(
+        address makerAddress,
+        bytes32 makerPoolId,
+        bytes32 poolId
```

Comment:
Do we need this field? The caller already passes in `poolId`, so it's already known.
Maybe we can even consolidate a bunch of these maker errors with an error code pattern if they share similar fields.